tests/server/test_compilerservice.py

```diff
@@ -450,10 +452,11 @@ def _compile_and_assert(env, export=True, meta={}, env_vars={}, update=False, ex
     # no export
     compile, stages = await _compile_and_assert(env=env, export=False)
     assert stages["Init"]["returncode"] == 0
+    assert stages["Creating venv"]["returncode"] == 0
```

Comment:
Isn't it rather confusing to always have a compiler stage called  `creating venv`, even if no venv has to be created? Should we call it `Ensure venv` or something similar?

Reply:
I went for venv check, as ensure is not a super clear or common word I think.